ert3/dummy_evaluator.py

```diff
@@ -0,0 +1,9 @@
+def dummy_evaluator(coefficients):
+    data = []
+    for (a, b, c) in coefficients:
+        data.append(
+            {
+                "polynomial_output": [a * x ** 2 + b * x + c for x in range(10)],
+            }
+        )
+    return data
```

Comment:
This is nowhere near the interface of the real evaluator, but I guess that's not the point?

Reply:
No, the point is just to get the evaluation out of `ert3.console`, connect it with the dummy `storage` and then reiterate from there :)